apps/admin/frontend/src/screens/reports_screen.tsx

```diff
@@ -41,19 +41,22 @@ export function ReportsScreen(): JSX.Element {
     converter,
     isOfficialResults,
     isTabulationRunning,
-    generateExportableTallies,
     fullElectionTally,
     fullElectionManualTally,
     configuredAt,
     logger,
     auth,
   } = useContext(AppContext);
-  const castVoteRecordFileModeQuery = getCastVoteRecordFileMode.useQuery();
   assert(isElectionManagerAuth(auth));
   const userRole = auth.user.role;
   assert(electionDefinition && typeof configuredAt === 'string');
   const { election } = electionDefinition;
 
+  const castVoteRecordFileModeQuery = getCastVoteRecordFileMode.useQuery();
+  const semsExportableTalliesQuery = getSemsExportableTallies.useQuery(
+    converter === 'ms-sems'
```

Comment:
nit: This was a little confusing at first (had to read the code a few times and the react-query documentation) - could the `enabled` param be renames to `fetchImmediately` or something obvious (or better yet, make it an object param, so that this reads something like this instead:
```
useQuery({ fetchImmediately: converter === 'ms-sems' })
```